src/scss/skin-modern/components/_subtitlesettingspaneltogglebutton.scss

```diff
@@ -0,0 +1,11 @@
+// demo for extracted audio tracks and subtitle settings from the settings panel direct into the controlBar
+@import '../variables';
```

Comment:
Again there is no equivalent `Component` class. In this case, you can create a `SubtitleSettingsPanelToggleButton extends SettingsPanelToggleButton`.

Reply:
I had one but i was not sure because the only different to the `SettingsPanelToggleButton` was a different `cssClass`.

Reply:
Yes both is not really ideal. Ok then please leave the file in its place but extend the `ui-settingstogglebutton` instead of `ui-button`.